<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServeurMonoWidget</class>
 <widget class="QWidget" name="ServeurMonoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ServeurMonoWidget</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>681</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_Messages">
    <item>
     <widget class="QLabel" name="label_Messages">
      <property name="text">
       <string>Messages </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit_Messages"/>
    </item>
    <item>
     <widget class="QLabel" name="label_Etat">
      <property name="text">
       <string>Ã‰tat</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit_Etat"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Quitter">
      <property name="text">
       <string>Quitter</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>681</width>
     <height>56</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_Port">
      <item>
       <widget class="QLabel" name="label_Port">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_Port">
        <property name="minimum">
         <number>1025</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>8888</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Lancer">
        <property name="text">
         <string>Lancer</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Quitter</sender>
   <signal>clicked()</signal>
   <receiver>ServeurMonoWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>688</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>641</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
